common/jenkins-agents/nodejs18/docker/Dockerfile.ubi8

```diff
@@ -0,0 +1,62 @@
+FROM opendevstackorg/ods-jenkins-agent-base-ubi8:latest
+
+# Labels consumed by Red Hat build service
+LABEL com.redhat.component="jenkins-agent-nodejs-18-rhel7-container" \
```

Comment:
isn't it ubi8 instead of rhel7 ?

Reply:
Now as I look at this... the labels section definitely needs improvement. Regarding this topic there seems to be #506 for dealing with this on all agents. I would not fix it with this PR.

Reply:
I'd prefer to fix it now.